### ISSUE TITLE
Enhance VRT with zoomed text-only screenshot variant of components

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2253]
Scope
---
Due to accessibility compliance, all PDS components must support Browser text-only zoom up to 200% without breaking behavior, functionality or visual experience. To automate this process, special VRT's should be created which test against text-zooming issues.

Notes  
---
- Research which framework (Puppeteer or Playwright) is best for testing this behavior
- https://github.com/puppeteer/puppeteer/issues/1157#issuecomment-453268101
- Use a single test spec file which navigates to all component pages with resolution `1000px` and execute the vrt test 
  
Acceptance criteria
---
- [x] VRT's for all components (starting with radio-button and checkbox) are created to test against text-only zoom issues

Subtasks
---
- [x] Task
